core/views/browse.py

```diff
@@ -196,6 +205,23 @@ def page(request, lccn, date, edition, sequence):
         else:
             explanation = ""
 
+    # if no word highlights were requests, see if the user came
+    # from search engine results and attempt to highlight words from their
+    # query by redirecting to a url that has the highlighted words in it
+    if not words:
```

Comment:
Now that the words redirect is in the other view, isn't `words` undefined? My editor shows this `flake8` output:

```
core/views/browse.py:195:1: E302 expected 2 blank lines, found 1
core/views/browse.py:211:12: F821 undefined name 'words'
core/views/browse.py:216:87: F821 undefined name 'path_parts'
```